bcp47/tests/Data/BCP47/TrieSpec.hs

```diff
@@ -6,9 +8,14 @@ import Prelude hiding (lookup)
 
 import Data.BCP47
 import Data.BCP47.Trie
+import Data.BCP47.Trie.Internal (mapMaybe)
```

Comment:
We shouldn't be importing internal where we don't have to.

Reply:
whoops!